sensorhub-android-app/src/org/sensorhub/android/MainActivity.java

```diff
@@ -448,20 +448,17 @@ public void onClick(DialogInterface dialog, int whichButton)
                 updateConfig(PreferenceManager.getDefaultSharedPreferences(MainActivity.this), runName);
 
                 AndroidSensorsConfig androidSensorConfig = (AndroidSensorsConfig) sensorhubConfig.get("ANDROID_SENSORS");
-                Log.d("OSH-DEBUG", androidSensorConfig.toString());
                 VideoEncoderConfig videoConfig = androidSensorConfig.videoConfig;
 
-                if(videoConfig.selectedPreset >= 0 && videoConfig.selectedPreset < videoConfig.presets.length){
-                    Log.d("OSH-DEBUG", "TEST MESSAGE...");
```

Comment:
Can you either remove or improve the 2 debug messages?

Reply:
They are removed, I think? The actual file doesn't contain them.

Reply:
Oh yes, sorry I didn't look at it correctly. Merging now